router/handler_link/bind.go

```diff
@@ -166,11 +164,19 @@ func (self *heartbeatCallback) HeartbeatRespRx(ts int64) {
 func (self *heartbeatCallback) CheckHeartBeat() {
 	log := pfxlog.Logger().WithField("channelId", self.ch.Label())
 	now := time.Now().UnixMilli()
-	if self.firstSent != 0 && (now-self.firstSent > 30000) && (now-self.lastResponse > 30000) {
+	if delta := now - self.lastResponse; delta > 30000 {
```

Comment:
is 30000 not the configurable value?

Reply:
I'll add this in a second pass, but it requires more plumbing b/c the default heartbeat options type doesn't have a space for it, so I need to create a custom object and thread it through.